Example/Alps/AppDelegate.swift

```diff
@@ -20,57 +20,56 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         
-        MatchMore.apiKey = "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJpc3MiOiJhbHBzIiwic3ViIjoiNDk4YWQ2ZWMtNjA4Ny00YTdjLThkNzEtZTA4MjhjNzQzZjJkIiwiYXVkIjpbIlB1YmxpYyJdLCJuYmYiOjE1MTA3NzY1ODQsImlhdCI6MTUxMDc3NjU4NCwianRpIjoiMSJ9.jd6nQzrB32mR0a_DR2keb6GS6YqHbphYJ5_RczVopaq5D8Y7L4bO29ObHcZD8aYbZQQUooJOC0D6EduXluozHw"
-        MatchMore.worldId = "498ad6ec-6087-4a7c-8d71-e0828c743f2d"
+        MatchMore.apiKey = "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJpc3MiOiJhbHBzIiwic3ViIjoiYzM5MzFhNDgtYmQ4Mi00NDVmLWI2NTYtMTEyN2ZkY2FiYjBlIiwiYXVkIjpbIlB1YmxpYyJdLCJuYmYiOjE1MTExODMxOTgsImlhdCI6MTUxMTE4MzE5OCwianRpIjoiMSJ9.ZvZ-cWwlUJv_dPpn1pSUoHoRT-7yoH4HjFqofnaDxMk5ZSwh0v9yn2HmnxejixinApGr-P-PAXcbisFuREVgPA"
+        MatchMore.worldId = "c3931a48-bd82-445f-b656-1127fdcabb0e"
         
-        MatchMore.createMainDevice { result in
+        MatchMore.startUsingMainDevice { result in
             guard case .success(let mainDevice) = result else { print(result.errorMessage ?? ""); return }
-            print("Device was created \(mainDevice.encodeToJSON())")
+            print("🏔 Using device: 🏔\n\(mainDevice.encodeToJSON())")
```

Comment:
We keep the print or NSLog ?  Cause I am using NSLog more than print

Reply:
I think print is simpler and more swifty.

Though it appears that NSLog is slower:
https://stackoverflow.com/questions/25951195/swift-print-vs-println-vs-nslog